packages/api/ecs/async-operation/package.json

```diff
@@ -25,7 +25,7 @@
     "@cumulus/db": "18.2.0",
     "@cumulus/es-client": "18.2.0",
     "@cumulus/logger": "18.2.0",
-    "aws-sdk": "^2.1492.0",
+    "@aws-sdk/client-lambda": "^3.529.1",
```

Comment:
ohh nice 😄. We should definitely start removing aws-sdk when we spot it and it's no longer used in the package.